src/styles/app.css

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+.App {
```

Comment:
You should use kebab-case for this classname, no capital letters as you are with all the others